cpp/generator.cpp

```diff
@@ -120,9 +119,9 @@ std::string detokenize(const std::vector<int64_t> &tokens) {
     wcstombs(buf, ws.c_str(), l + 1);
     std::string s(buf);
     delete[] buf;
-    return s;
+    return {s};
```

Comment:
Why do we need the `{}`?

Reply:
Oh that's incorrect. I once turned it to a pair and forgot to turn it back. Let me delete that.